addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py

```diff
@@ -17,7 +17,7 @@ class TimesheetAttendance(models.Model):
 
     @api.model_cr
     def init(self):
-        self._cr.execute("DROP VIEW IF EXISTS hr_timesheet_attendance_report")
+        self._cr.execute("DROP VIEW IF EXISTS %s" % self._table)
```

Comment:
why not call `drop_view_if_exists(self._table)` from `odoo.tools.sql`?

Reply:
Thank you, good idea, now it is done